src/index.html

```diff
@@ -11,6 +11,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form method="post" data-qa="big" >
+      <label class="big">
+        <img class="bigImage" src="images/Search.svg" alt="searchSign">
+        <input 
+          type="text" 
+          class="bigText" 
```

Comment:
```suggestion
          class="text bigText" 
```
В тебе дублюється багато стилів у класах bigText та smallText.
Створи один загальний клас зі спільними стилями для обох інпутів і два окремих класа зі специфічними стилями для кожного